src/experiments.js

```diff
@@ -189,9 +197,16 @@ export function experimentTogglesOrNull(win) {
  * @param {!Window} win
  * @return {!Object<string, boolean>}
  */
-function getExperimentTogglesFromCookie(win) {
-  const experimentCookie = getCookie(win, COOKIE_NAME);
-  const tokens = experimentCookie ? experimentCookie.split(/\s*,\s*/g) : [];
+function getExperimentToggles(win) {
+  let experimentsString = '';
+  if ('localStorage' in win) {
```

Comment:
I think even testing `in` causes an error in Safari. Let's move this inside the try-catch.